fazel_file_generator/fazel_file_creator.py

```diff
@@ -154,11 +155,12 @@ def main(args=None):
             if not args.quiet:
                 print("Source not found in common list, searching online")
             source = SkyCoord.from_name(args.source)
-            source_found = False
+            source_found = True
```

Comment:
```suggestion
            source_found = False
```

Reply:
Rik is right here: `source_found` strictly means whether or not the source was in the hard-coded source list for off-line work.

But since `source_found` is already set to `False` on top, this line can be removed.